cmd/uql/execute.go

```diff
@@ -143,11 +144,34 @@ func (b defaultBackend) Continue(link *Link) (parsedResponse, error) {
 	}, nil
 }
 
-var backend uqlService = &defaultBackend{}
+type UqlClient interface {
+	ExecuteQuery(query *Query, apiVersion ApiVersion) (*Response, error)
+	ContinueQuery(dataSet *DataSet, rel string) (*Response, error)
+}
+
+type DefaultClient struct {
+	Backend uqlService
+}
+
+func (c DefaultClient) ExecuteQuery(query *Query, apiVersion ApiVersion) (*Response, error) {
+	if c.Backend == nil {
+		return nil, fmt.Errorf("uql Backend missing")
+	}
+	return executeUqlQuery(query, apiVersion, c.Backend)
+}
+
+func (c DefaultClient) ContinueQuery(dataSet *DataSet, rel string) (*Response, error) {
+	if c.Backend == nil {
+		return nil, fmt.Errorf("uql Backend missing")
+	}
+	return continueUqlQuery(dataSet, rel, c.Backend)
+}
+
+var client UqlClient = &DefaultClient{Backend: &DefaultBackend{}}
 
 // ExecuteQuery sends an execute request to the UQL service
 func ExecuteQuery(query *Query, apiVersion ApiVersion) (*Response, error) {
```

Comment:
as aforementioned, delete this and `ContinueQuery` method. Please, move the comments to the client interface